webviewer/src/pages/Main/PageMain.jsx

```diff
@@ -6,22 +6,21 @@ import { AreasSidebar } from 'components/Areas';
 import { Map } from 'components/Map';
 import { ContactUs } from 'components/ContactUs';
 
-import { useAreasActions, useUserActions } from 'state';
+import { useAreasActions } from 'state';
 
 export const PageMain = ({ ...props }) => {
-  const { getCurrentUser } = useUserActions();
   const { getAreas } = useAreasActions();
+  const isOpen = props.history.action === 'PUSH' && props.location.state?.isOpenSidebar;
```

Comment:
useMemo?

Reply:
I don't see the point. It's will not improve performance, but will only add an extra wrapper function